client/views/windows/settings.tpl

```diff
@@ -14,6 +14,11 @@
 	</div>
 
 	<div class="row">
+		<div class="col-sm-12" id="webAppInstallButton" style="display:none">
```

Comment:
Class instead of inline style?

Reply:
Not sure how I feel about this, as it should always be display none, correct?

Reply:
We already use this method if we display something with jQuery.

Ideally we'd use one global class (like `.shown`) or `hidden` attribute.

Can we leave this for a future PR since this PR doesn't add any styles at all?

Reply:
Or `hidden` CSS class? 🤷‍♂️